.gitignore

```
@@ -1,6 +1,7 @@
 .DS_Store
 
-.env*
+# .env*
```

Ich habe lokal noch welche liegen für Testumgebungen, das ist vielleicht nicht perfekt, aber ich würde lieber alle auf die `gitignore` setzen und einzelne explizit erlauben. 
```suggestion
.env*
!.env.sample
```

---

kann man so machen; ich würd's halt umdrehen, dann ist das alles in der Versionsverwaltung; wenns fertig ist, kann man die "unnötigen" Files immer noch entfernen. 

Damit beim Deployment nicht zuviel Schrott mitkommt, müßte man mehr "maschinelle Intelligenz" reinstecken, z.B.: nur .env.${hostname} deployen, oder eine .env_.template_ definieren und die _echten_ Werte bei der Installation aus irgendwelchen Properties lesen, ....